practice_app/pages/api/getTranslationModule.js

```diff
@@ -0,0 +1,803 @@
+exports.checkTranslationResponse = function (response, sourceLanguage, targetLanguage, originalText) {
+    const translation = response.data[0][0][0]
+    const sourceText = response.data[0][0][1]
+    const sourceLang = getSourceLangName(response.data[2])
+    const isAlternativeSourceExist = response.data[8] ? response.data[8][0].length > 1 : false
+
+    const alternativeSourceLangs = []
+    if (isAlternativeSourceExist) {
+        let i
+        for (i = 0; i < response.data[8][0].length; i++) {
+            alternativeSourceLangs.push(getSourceLangName(response.data[8][0][i]))
+        }
+    }
+
+    return {
+        translation,
+        sourceLang,
+        isAlternativeSourceExist,
+        alternativeSourceLangs,
+        sourceText,
+        originalText,
+        sourceLanguage,
+        targetLanguage,
+    }
+}
+
+var getSourceLangName = (exports.getSourceLangName = function (sl) {
+    let sourceLangName = 'lang_not_found'
+    switch (sl) {
+        case 'en': {
+            sourceLangName = 'English'
+            break
+        }
+        case 'es': {
+            sourceLangName = 'Spanish'
+            break
+        }
+        case 'tr': {
+            sourceLangName = 'Turkish'
+            break
+        }
+        case 'de': {
+            sourceLangName = 'German'
+            break
+        }
+        case 'fr': {
+            sourceLangName = 'French'
+            break
+        }
+        case 'it': {
+            sourceLangName = 'Italian'
+            break
+        }
+        case 'fi': {
+            sourceLangName = 'Finnish'
+            break
+        }
```

Comment:
No need for these specific cases since they can be handled by the "default" case. I think you should remove the `switch`  completely.

Reply:
Removed.